documentation/configuration.markdown

```diff
@@ -137,6 +137,14 @@ In a default installation, this file doesn't exist. You'll need to create it if
     This setting is used to enable [external SSL termination.](./external_ssl_termination.markdown) If enabled, Puppet Server will ignore any actual certificate presented to the Jetty webserver, and will rely completely on header data to authorize requests. This is very dangerous unless you've secured your network to prevent any untrusted access to Puppet Server.
 
     You can change Puppet's `ssl_client_verify_header` setting to use another header name instead of `X-Client-Verify`; the `ssl_client_header` setting can rename `X-Client-CN`. The `X-Client-Cert` header can't be renamed.
+    
+    Note that this setting only applies to HTTP endpoints served by the "master"
+    service.  The applicable endpoints include those listed in the 
+    "Configuration Management Services", "Informational Services", and
+    "V2 HTTP API" sections of the [Puppet HTTP API](http://docs.puppetlabs.com/references/3.7.latest/developer/file.http_api_index.html)
```

Comment:
Thanks, @nfagerlund.  I updated the PR in 7707d07 to point at the 3.8.1 link that you listed.

FYI, in case you hadn't seen this recently, I noticed that the endpoint references under the sections on the github page don't function as links, e.g., "{file:api/docs/http_catalog.md Catalog}".  Not sure if that's intentional?